### ISSUE TITLE
Cron not working & errors in journal

### ISSUE THREAD

**Issue by Robin-Abibia** [ansible-semaphore/semaphore#1516]
We use Semaphore for automation processes, like continous deployment of web projects, backup of databases, etc. on a debian system. But when there are overlapping cron-jobs, the scheduler doesn't work anymore, so no tasks are automatically fired. When this happens, there are some errors in the journal:

`Sep 26 08:03:15 vm-ansible-00 semaphore[1891914]: time="2023-09-26T08:03:15Z" level=error msg="websocket: close 1006 (abnormal closure): unexpected EOF" level=Error`
`Sep 26 08:03:15 vm-ansible-00 semaphore[1891914]: time="2023-09-26T08:03:15Z" level=error msg="write tcp 127.0.0.1:3000->127.0.0.1:51130: use of closed network connection" level=Error`
`Sep 26 08:03:15 vm-ansible-00 semaphore[1891914]: time="2023-09-26T08:03:15Z" level=error msg="close tcp 127.0.0.1:3000->127.0.0.1:51130: use of closed network connection" level=Error`

We use Ansible-Semaphore v2.9.18, but with 2.9.4 this also occurs. The scheduling of tasks is very important to us. If we schedule using crontab, there is no need for Semaphore anymore.

**Comment by fiftin** [comment]
Hi @Robin-Abibia it is a bug. Will be fixed ASAP. Thank you.

**Comment by fiftin** [comment]
@Robin-Abibia could you please provide more details by the issue? It is critical bug.

**Comment by fiftin** [comment]
@Robin-Abibia Do you have similar?

<img width="400" alt="image" src="https://github.com/ansible-semaphore/semaphore/assets/914224/b983f1d8-c7ca-4d7c-a3c7-c03a903e3f0a">

Looks like is concurrent issue.

**Comment by Robin-Abibia** [comment]
No, when this occurs, the tasks don't even show in the UI. Only fix is to remove all cron jobs, after that, it works again.

**Comment by juju4** [comment]
Also observing this kind of errors
From time to time, on my semaphore instance (2.12.4-c0551d5-1738565637, ubuntu 24.04, setup with https://github.com/juju4/ansible-semaphoreui/)
```
Mar 01 10:12:34 mlsemaphore004 semaphore[17424]: time="2025-03-01T10:12:34Z" level=error msg="websocket: close 1006 (abnormal closure): unexpected EOF" fields.level=Error
Mar 01 10:12:34 mlsemaphore004 semaphore[17424]: time="2025-03-01T10:12:34Z" level=error msg="write tcp 127.0.0.1:3000->127.0.0.1:59078: use of closed network connection" error="Cannot send close message"
```
Any way to trace back and avoid them?
